recipes/fedfred/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "fedfred" %}
+{% set version = "1.2.9" %}
+{% set python_min = "3.9" %}
```

Comment:
You need to set this only if the package min Python is less than conda-forge's default.

```suggestion
```